### PR TITLE
Cldmgr unique secret names

### LINKED ISSUE #3480: [MCG] cloud manager uses static name for secrets
cloud manager should create secrets with unique names in case another entity is created. 
if not changed this will cause a failure since its currently hard coded

### PR DESCRIPTION
**Pull request by bernerhat**
Fixes #3480

### CODE REVIEW

File: ocs_ci/helpers/helpers.py [hunk 1]
```diff
@@ -56,7 +57,8 @@ def create_unique_resource_name(resource_description, resource_type):
     Returns:
         str: A unique name
     """
-    return f"{resource_type}-{resource_description[:23]}-{uuid4().hex}"
+    name = f"{resource_type}-{resource_description[:23]}-{uuid4().hex}"
+    return name if len(name) < 64 else name[:63]
```
Comment (MeridianExplorer) [inline ocs_ci/helpers/helpers.py:61]: I think it's too risky to leave the length so close to the maximum possible.
IMO, 40 characters should be sufficient, and it'll also help keep naming more clean and digestible

**Comment by MeridianExplorer** [comment]
[Source regarding the name length limit](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names)

**Review by MeridianExplorer — APPROVED** [review]

**Review by ebenahar — DISMISSED** [review]
@bernerhat Please run tier1 for validation. This PR touches a sensitive area

**Review by ebenahar — APPROVED** [review]

**Comment by bernerhat** [comment]
> PR validation on existing cluster
> 
> Cluster Name: aberner-dec9
> Cluster Configuration:
> PR Test Suite: tier1
> PR Test Path: tests/manage/mcg/
> Additional Test Params:
> OCP VERSION: 4.6
> OCS VERSION: 4.6
> 
> [Job FAILED](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/3545//testReport) (installation failed, tests not executed).

test_scaling_under_load_ocs_logs failed due to maintenance work that occurred during the execution not due to the pr changes

test_multipart_upload_operations_ocs_logs failed due to the open [bug](https://bugzilla.redhat.com/show_bug.cgi?id=1901442) and not due to the pr changes as well.

Therefore marking this as verified

**Review by MeridianExplorer — APPROVED** [review]

**Comment by bernerhat** [comment]
> PR validation on existing cluster
> 
> Cluster Name: aberner-dec9
> Cluster Configuration:
> PR Test Suite: tier1
> PR Test Path: tests/
> Additional Test Params:
> OCP VERSION: 4.6
> OCS VERSION: 4.6
> 
> [Job UNSTABLE](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/3555//testReport) (some or all tests failed).

failures in this run are unrelated to the pr, therefore verified over all tests/

**Pull request merged**